packages/api/test/unit/client/httpClient.test.ts

```diff
@@ -128,6 +128,7 @@ describe("httpClient json client", () => {
       return Promise.reject(Error("did not throw"));
     } catch (e) {
       if (!(e instanceof HttpError)) throw Error(`Not an HttpError: ${(e as Error).message}`);
+      // Discuss error handling: https://github.com/ChainSafe/lodestar/issues/5359#issuecomment-1529027981
       expect(e.message).to.equal("Service Unavailable: Node is syncing");
```

Comment:
Reference: https://github.com/ChainSafe/lodestar/issues/5359#issuecomment-1529027981
As @nflaig mentioned in the comment, we should remove the stacktrace here, log as `warn` and consider logging something that is expected like "Node is currently syncing..."

Reply:
This is a test, the actual logger statement to change should be somewhere else

Reply:
There are no more verbose "Node is syncing" error logs on the VC side, those are all now logged as `warn` (single line log)

Related PRs
- https://github.com/ChainSafe/lodestar/pull/5467
- https://github.com/ChainSafe/lodestar/pull/5511